packages/skill-lesson/lib/products.ts

```diff
@@ -47,7 +47,26 @@ export const getProductModules = async (productIds: string[]) => {
       productIds,
     },
   )
-  return product
+  return products
+}
+
+export const getModuleProducts = async (productIds: string[]) => {
+  const products = await sanityClient.fetch(
+    groq`*[_type == "module" && moduleType == 'workshop' && !(null in resources[].productId)] | order(_createdAt desc) {
+    _id,
+    resources,
+    "productId": resources[@._type == 'product'][0].productId,
+    }`,
+    {
+      productIds,
+    },
+  )
+  return products.map((product: any) => {
+    // product can also be a module with product resource
+    // TODO: adjust logic in here: https://github.com/skillrecordings/products/blob/main/packages/skill-lesson/utils/ability.ts#L72
+    // so that this object doesn't have to be reshaped
+    return {...product, modules: [product]}
+  })
```

Comment:
this query will get workshops with product resource which is what we have on PT. I had to reshape the object a little bit because of the logic here which is expecting `modules` to be present: https://github.com/skillrecordings/products/blob/main/packages/skill-lesson/utils/ability.ts#L72

should likely be improved, but works fine and will enable users to view content again.